bindsnet/encoding/__init__.py

```diff
@@ -1,9 +1,73 @@
+import pdb
 import torch
 import numpy as np
-
+import math
 from typing import Optional, Union, Iterable, Iterator
 
 
+def timing(datum: torch.Tensor, time: int = None, max_delay: int = 10,
```

Comment:
We have a method called `rank_order_encoding` which seems to do the same as this function.

Reply:
You should remove this method, since `rank_order_encoding` already exists.